src/actions/types.ts

```diff
@@ -119,7 +118,7 @@ export type PanelComponentProps = {
   appState: AppState;
   updateData: (formData?: any) => void;
   appProps: ExcalidrawProps;
-  data?: Partial<{ id: string; size: ToolButtonSize }>;
+  data?: Record<string, any>;
```

Comment:
couldn't figure out how to selectively set data type for specific actions, so for now we'll have a less safe type